web/models/summary_helper.ex

```diff
@@ -0,0 +1,80 @@
+defmodule CoursePlanner.SummaryHelper do
+  @moduledoc """
+    Provides helper functions for the summary page
+  """
+  use CoursePlanner.Web, :model
+
+  alias CoursePlanner.{Repo, Terms.Term, OfferedCourse}
+
+  def get_term_offered_course_for_user(%{id: user_id, role: role}) do
```

Comment:
aha, haven't thought of "enable us to easily query the summary for yesterday or tomorrow", nice point. will do it